manifests/transport.pp

```diff
@@ -1,14 +1,12 @@
 # @summary Configures Sensu transport
 #
-# Configure Sensu Transport
+# Configures Sensu transport
 #
-class sensu::transport {
-
-  if $::sensu::transport_type != 'redis' {
-    $ensure = 'absent'
-  } else {
-    $ensure = 'present'
-  }
+# @param ensure If to create or remove the transport.json file
+#
+class sensu::transport (
+  Enum['present','absent'] $ensure = 'present',
```

Comment:
Since we're adding a param, this would need the param to be documented as well as tests, unless I'm missing something here.

Reply:
Actually I tried to make tests, but it's a bit tricky, if not impossible, to test directly the sensu::transport class, as all the other classes which are declared in the main class and use it as entry point to parameters.
To test it's behaviour we should pass via the main class (https://github.com/sensu/sensu-puppet/blob/master/spec/classes/sensu_transport_spec.rb#L4), and this would have required to add an additional parameters like sensu::transport_ensure, which seemed too much for me.

Reply:
@Phil-Friderici Any thoughts?

Reply:
ping @Phil-Friderici